inst/components/scss/grid.scss

```diff
@@ -3,13 +3,19 @@
   gap: var(--bslib-spacer, 1rem);
   height: var(--bslib-grid-height);
   &.grid {
+    // Don't let intrinsic width of a column affect the width of grid cols
+    grid-template-columns: repeat(var(--bs-columns, 12), minmax(0, 1fr));
```

Comment:
Might be worth adding a comment that this overrides Bootstrap's `repeat(var(--bs-columns, 12), 1fr);` default